ratis-docs/src/site/markdown/configurations.md

```diff
@@ -458,11 +458,11 @@ the leader won't send snapshots to follower.
 It will just send a notification to that follower instead.
 The follower's statemachine is responsible for fetching and installing snapshot by some other means.
 
-| **Property**    | `raft.server.log.appender.wait-time.min`       |
-|:----------------|:-----------------------------------------------|
-| **Description** | wait time between two subsequent AppendEntries |
-| **Type**        | TimeDuration                                   |
-| **Default**     | 10ms                                           |
+| **Property**    | `raft.server.log.appender.wait-time.min`                                   |
+|:----------------|:---------------------------------------------------------------------------|
+| **Description** | wait time between two subsequent AppendEntries. Must be a positive number. |
```

Comment:
@jojochuang , thanks a lot for working on this!

This conf has two purposes:
1.  wait time between two subsequent AppendEntries, and
2. set to waitForReady in `GrpcLogAppender` (for the sleep waiting a gRPC stream to ready).

For (1), zero is allowed.  For (2), zero becomes 1ms.

Do you think that the usage in (2) is problematic?   If yes, we may use a different conf.